docs/Client_Server.md

```diff
@@ -0,0 +1,39 @@
+# Client and Server
+
+## Precompile
+
+Server side we precompile all our React components into static HTML.
+
+Use the `npm install` command to trigger or update the precompile.
+
+### Webpack
+
+Webpack creates static HTML pages from a list of react routes.
```

Comment:
I think you might want to be specific here and mention about Webpack plugin because Webpack doesn't do this by default, so it might confused people.